donkeycar/utilities/TrackSpeedPlanner/.gitignore

```diff
@@ -0,0 +1,45 @@
+# Python
```

Comment:
Can this be trimmed down to the relevant entries only?

Reply:
I think we don't want to ship any .gitignore in some subfolder. We have a .gitignore at the root level, and I don't think that any of the .gitignore content here would suppress a file that is generated from your application, so it would be cleaner to remove it from the PR.